CacheMoney/backend/src/main/java/com/revature/cachemoney/backend/beans/models/Notification.java

```diff
@@ -0,0 +1,32 @@
+package com.revature.cachemoney.backend.beans.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+/**
+ * @author Mika Nelson, Dylan Wilson, Cullen Kuch, Max Hilken, Tyler Seufert
+ */
+@NoArgsConstructor
+@Data
+@AllArgsConstructor
+@Entity
+@Table(name = "Notifications")
+public class Notification {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column
+    private int notif_id;
+    @Column
+    private String subject;
+    @Column
+    private String notif_text;
+    @Column
+    private int user_id;
```

Comment:
Should this be a ManyToOne?

Reply:
We had issues trying to implement the relationship, but in a perfect world, yes.

Reply:
Can I ask what went wrong?